framework/tests/all-systems/Enums.java

```diff
@@ -13,18 +14,18 @@ void foo2(MyEnumSet<ElementType> p) {}
 
   <E extends Enum<E>> void mtv(Class<E> p) {}
 
-  <T extends Object> T checkNotNull(T ref) { return ref; }
```

Comment:
I don't understand what you mean with this.
You're removing a test from all-systems. If that test isn't working with the Lock Checker, you should only ignore it for that one checker, not for all of them.